keytest.py

```diff
@@ -0,0 +1,6 @@
+from pykob import kob
+
+myKOB = kob.KOB(port="com3")
+myKOB.setSounder(False)
+while True:
+    print(myKOB.key())
```

Comment:
Can we do like I did in 'Closer-Test.py' and use the 'config' module to get the port?

```
Closer-Test.py:
import sys
from pykob import kob
from pykob import config

try:
    interface = config.serial_port

    myKOB = kob.KOB(port=interface)
    myKOB.setSounder(False)
    while True:
        print(myKOB.key())
except KeyboardInterrupt:
    print()
    print("Thank you for using the Closer-Test!")
    sys.exit(0)     # Indicate this was a normal exit
    
```
Looking at it, they seem to be doing the same thing. I checked 'Closer-Test.py' in from the code I got from you when you asked if I could test on my Mac and Linux machines.

Reply:
I did this because the Mac and Linux don't have 'com3' and my Windows machine uses 'com5'. I also wanted to break out without getting a back-trace.